protos/sources/vega/commands/v1/transaction.proto

```diff
@@ -9,37 +9,19 @@ import "vega/commands/v1/validator_commands.proto";
 
 option go_package = "code.vegaprotocol.io/vega/protos/vega/commands/v1";
 
+// The input data for a transaction containing a network command for the Vega network to execute.
+// Once populated the protobuf message should be marshalled into a byte array and included in a transcation message.
 message InputData {
   // This was the old restore checkpoint command
   reserved 4001;
-  // Number to provide uniqueness to prevent accidental replays and,
-  // in combination with `block_height`, deliberate attacks.
-  // A nonce provides uniqueness for otherwise identical transactions,
-  // ensuring that the transaction hash uniquely identifies a specific transaction.
-  // Granted all other fields are equal, the nonce can either be a counter
-  // or generated at random to submit multiple transactions within the same
-  // block (see below), without being identified as replays.
-  // Please note that Protocol Buffers do not have a canonical, unique encoding
-  // and therefore different libraries or binaries may encode the same message
-  // slightly differently, causing a different hash.
+  // Random number to provide uniqueness for otherwise identical transactions.
   uint64 nonce = 1;
-
-  // Block height at which the transaction was made.
-  // This should be the current block height. The transaction will be valid
-  // from the block and up to the `tolerance` block height.
-  // Example: If the network has a tolerance of 150 blocks and `block_height`
-  // is set to `200`, then the transaction will be valid until block `350`.
-  // Note that a `block_height` that is ahead of the real block height will be
-  // rejected. The tolerance can be queried from the chain's network parameters.
-  // `block_height` prevents replay attacks in conjunction with `nonce` (see above).
```

Comment:
Not sure why remove all this? for someone trying to build the transaction themselves this can be useful for people implementing the transaction format themselves.
Not sure it's needed too tho, it's not super difficult to get what it is, maybe just the block_height to undersand how it's used.

Reply:
I know, but its just too much information to have here. If we want to help people construct a transaction without using the vegawallet we should write a docs page similar to:
https://docs.vega.xyz/testnet/tools/vega-wallet/cli-wallet/latest/guides/build-send-transactions

We also have a docs page on proof-of-work: https://docs.vega.xyz/testnet/concepts/vega-chain/network#proof-of-work 

where we can put lots of in-depth details about the life-time of proof-of-works, and how to generate it by calling `LastBlockHeightWhatever` to get the block-hash details and that.

Reply:
fair enough.